bin/node/runtime/src/lib.rs

```diff
@@ -1017,6 +1027,8 @@ impl pallet_elections_phragmen::Config for Runtime {
 	type TermDuration = TermDuration;
 	type MaxVoters = MaxVoters;
 	type MaxCandidates = MaxCandidates;
+	type MaxMembers = MaxMembers;
+	type MaxRunnersUp = MaxRunnersUp;
```

Comment:
We should not need these two either. 

Any vector that needs to hold members is already bounded by `DesiredMembers`.
Any vector that needs to hold runners-up is bounded `DesiredRunnersUo`.

So need for these.